### DISCUSSION TITLE
Running on MacOS

### DISCUSSION THREAD

**Discussion by R3ck1e** [VoxelCubes/PanelCleaner#25] [General]
Can't run on Mac. Full trace:
Last login: Tue Nov  7 18:57:25 on ttys001
/Users/romangoncarov/Library/Python/3.9/bin/pcleaner ; exit;                    
romangoncarov@mbp-roman ~ % /Users/romangoncarov/Library/Python/3.9/bin/pcleaner ; exit;
/Users/romangoncarov/Library/Python/3.9/lib/python/site-packages/urllib3/__init__.py:34: NotOpenSSLWarning: urllib3 v2.0 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with 'LibreSSL 2.8.3'. See: https://github.com/urllib3/urllib3/issues/3020
  warnings.warn(
Traceback (most recent call last):
  File "/Users/romangoncarov/Library/Python/3.9/bin/pcleaner", line 5, in <module>
    from pcleaner.main import main
  File "/Users/romangoncarov/Library/Python/3.9/lib/python/site-packages/pcleaner/main.py", line 107, in <module>
    from docopt import magic_docopt
ImportError: cannot import name 'magic_docopt' from 'docopt' (/Users/romangoncarov/Library/Python/3.9/lib/python/site-packages/docopt/__init__.py)

Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.

//
Can't figure out what's wrong. The same error on Python 3.12. pip version is 23.2.1

**Comment by VoxelCubes** [comment]
This belongs in issues, so I've moved it over there. Fortunately it's a relatively trivial fix I thought I had squashed, you may be running an old version.
